src/verify/verify.test.ts

```diff
@@ -258,7 +258,7 @@ describe("isValidVc", () => {
         fetch: mockedFetch,
       })
     ).rejects.toThrow(
-      `The VC service provider ${MOCK_VC.issuer} does not advertize for a verifier service in its .well-known/vc-configuration document`
+      `The VC service provider ${MOCK_VC.issuer} does not advertise for a verifier service in its .well-known/vc-configuration document`
```

Comment:
Should this be changed? I thought we were supposed to use US vs UK english? Maybe its just a setting in the spell check settings?

Reply:
Interesting. TIL "advertize" is correct in US English. I had never come across that spelling so thought this was a typo.